resources/views/model.blade.php

```diff
@@ -96,17 +96,16 @@ public function {{ $belongsToMany['related_table'] }}() {
 @endforeach
     @endif
     @endif
+    @if($fillable) @foreach($fillable as $f) @if($f == "created_by_admin_user_id")
 
-    @if($fillable)
-        @foreach($fillable as $f)
-            @if($f == "created_by_admin_user_id")
-                public function createdByAdminUser(){
-                    return $this->belongsTo(AdminUser::class);
-                }
+    public function createdByAdminUser(){
```

Comment:
I will recommend to use traits. Prepare trait in craftable, and just use it here.